File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day22ModeMazeTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day22ModeMazeTest {
    @Test
    public void checkReal() {
        Day22ModeMaze im = new Day22ModeMaze("/twenty_eighteen/Day22-ModeMaze-input.txt");

        assertEquals(9659, im.getPart1Answer().intValue());
        assertEquals(1043, im.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day22ModeMaze im = new Day22ModeMaze("/twenty_eighteen/Day22-ModeMaze-example#1.txt");

        assertEquals(114, im.getPart1Answer().intValue());
        assertEquals(45, im.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day22ModeMazeTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day22ModeMazeTest  extends FlatSpec with Matchers {
  "2018 Day 22 - Input" should "calculate answers" in {
    val dp = new Day22ModeMaze("/twenty_eighteen/Day22-ModeMaze-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day22ModeMaze("/twenty_eighteen/Day22-ModeMaze-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
